dod-advana/gamechanger-ml
# Feature/UOT-130237

Added the ability to stop threads that are running in the background, tracking which file is being downloaded from s3 and failed processes are now being tracked along with a message that can be shown

To test
The best endpoint I found to test it on running downloadCorpus.  This should return the process status which should now have the thread id in it. Otherwise you can also hit the getProcessStatus endpoint.  From there you can make a post request to stopProcess with the dict {"thread_id":(int of thread id), "process":(name of the process so we can also update it in redis)}
